integrations/dbt_bq/README.md

```diff
@@ -0,0 +1,76 @@
+## <span style='color:#ff5f27'> 👨🏻‍🏫 DBT Tutorial with BigQuery </span>
+
+### <span style='color:#ff5f27'> 🏡 Cluster setup </span>
+
+To begin with, it's necessary to setup the Dataproc cluster.
+
+You need to configure an external Spark cluster to be able to interact with the Hopsworks Feature Store.
```

Comment:
First, you need to setup a Dataproc (Spark) cluster that will run the Python model in our DBT workflow. The Python model will write to the online feature store in Hopsworks.